### ISSUE TITLE
a11y: missing aria labels on toggle buttons in view dialog

### ISSUE THREAD

**Issue by ronnymikalsen** [NationalLibraryOfNorway/ngx-mime#446]
**Describe the bug**
It is missing aria labels on toggle buttons in view dialog. It is hard to undersatnd what the different toggle buttons do

**To Reproduce**

1.  Start demo app
2.  Click on 'View' button in toolbar
3.  Check aria labels

**Expected behavior**
Aria labels that describe the state of the toggle buttons

**Which versions of NGX-Mime, Angular, Material, OS, TypeScript, browsers are affected?**
>=15.1
